src/components/PixelShifter/PixelShifter.js

```diff
@@ -1,6 +1,21 @@
-import styled from 'styled-components';
+// @flow
+import React from 'react';
 
-export default styled.div`
-  width: 100%;
-  transform: translate(${props => props.x || 0}px, ${props => props.y || 0}px);
-`;
+type Props = {
+  x?: number,
+  y?: number,
+  reason: string,
+  children: React$Node,
+};
+
+const PixelShifter = ({ x = 0, y = 0, reason, children }: Props) => (
+  <div
+    style={{
+      transform: `translate(${x}px, ${y}px)`,
```

Comment:
I noticed that you omitted `width: 100%`, was that intentional?

Reply:
Mhmm, good catch! I omitted it because I don't think it was doing anything. I forget why I added it, but block elements like divs should take up 100% width of the parent by default 🤔

I should give it a quick visual pass to make sure nothing's wonky before merging, but yeah I think we can omit it.